package/score-worker/src/main/java/org/openscore/schema/WorkerBeanDefinitionParser.java

```diff
@@ -57,7 +52,12 @@
 		put(ReflectionAdapterImpl.class, null);
         put(SessionDataHandlerImpl.class, "sessionDataHandler");
 		put(SynchronizationManagerImpl.class, null);
-		put(WorkerConfigurationServiceImpl.class, "workerConfiguration");
+        put(WorkerConfigurationServiceImpl.class, "workerConfiguration");
```

Comment:
regarding the schedulers I prefer that the consumer,that uses the data, would run the jobs and not score.